### PR TITLE
File based config

### PR DESCRIPTION
**Pull request by MojoJolo**
Settings are now loaded using the `data/settings.yml` file.

This is a fix for this issue: https://github.com/scrapinghub/dateparser/issues/54

### CODE REVIEW

File: data/settings.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+settings:
+    PREFER_DATES_FROM: 'current_period'
```
Comment (waqasshabbir) [inline data/settings.yaml]: Please merge master, it has a new configuration `SKIP_TOKENS`.

**Comment by waqasshabbir** [comment]
@MojoJolo please merge latest master, it has a new configuration `SKIP_TOKENS`.

**Comment by MojoJolo** [comment]
@waqasshabbir merged it.

File: dateparser/conf.py [hunk 2]
```diff
@@ -43,6 +47,17 @@ class Settings(object):
     SKIP_TOKENS = ['t']
```
Comment (waqasshabbir) [inline dateparser/conf.py]: why not move `SKIP_TOKENS` to `settings.yaml`?

**Comment by MojoJolo** [comment]
@waqasshabbir sorry, forgot to place it in settings.yaml

**Comment by waqasshabbir** [comment]
Thank you @MojoJolo!

**Pull request merged**